index.html

```diff
@@ -27,17 +27,119 @@ <h1>I'm Saningo Glad to see you</h1>
+                  <p>A daily selection of privately personalized reads, no accounts or sign-ups required</p>
+                  <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                  </ul>
+                  <button type="button">See Project</button>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="card-2"></div>
+              <div class="card-desc">
+                <h2>Muilti-Post Stories</h2>
+                <ul>
+                  <li>FACEBOOK</li>
+                  <li>Full stack Dev</li>
+                  <li>2023</li>
+                </ul>
+                <div class="desc">
+                  <p>A daily selection of privately personalized reads, no accounts or sign-ups required</p>
+                  <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                  </ul>
+                  <button type="button">See Project</button>
+                </div>
+              </div>
+          </div>
 
-          <div class="empty">
+          <div class="card">
+            <div class="card-3"></div>
+              <div class="card-desc">
+                  <div>
+                    <h2>Facebook 360</h2>
+                    <ul>
+                      <li>FACEBOOK</li>
+                      <li>Full stack DEV</li>
+                      <li>2023</li>
+                    </ul>
+                    <div class="desc">
+                      <p>A daily selection of privately personalized reads, no accounts or sign-ups required</p>
+                      <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                      </ul>
+                      <button type="button">See Project</button>
+                    </div>
+                  </div>
+              </div>
+          </div>
 
+          <div class="card">
+            <div class="card-4"></div>
+            <div class="card-desc">
+                <div>
+                  <h2>Facebook React</h2>
+                  <ul>
+                    <li>FACEBOOK</li>
+                    <li>Full stack DEV</li>
+                    <li>2023</li>
+                  </ul>
+                  <div class="desc">
+                    <p>A daily selection of privately personalized reads, no accounts or sign-ups required</p>
+                    <ul>
+                      <li>html</li>
+                      <li>css</li>
+                      <li>javascript</li>
+                    </ul>
+                    <button type="button">See Project</button>
+                  </div>
+                </div>
+            </div>
           </div>
+
+        </section>
+        <!-- About Me section -->
+        <section class="about-me">
+          <h1>About Me</h1>
```

Comment:
- [ ] Please use other `hx` tag instead of `h1` tag. As `h1` can only use once as the header of the website. Kindly fix this.

Reply:
Thank you I will definitely work on that